docs/Development-Guides/Data-Syncs.md

```diff
@@ -20,4 +20,6 @@ Response Header | Description
 For batch endpoints (`/employment`, `/individual`, `/pay-statement`) Finch syncs all of the data for the connection in one atomic operation. This ensures that all items in the batch response you receive are from the same data sync.
 
 ### Pay statement history
-Finch currently fetches pay statements from the start of two years from the sync time. For example, if a sync happens on 06/01/2022, pay statements from 01/01/2020 thru 06/01/2022 will be fetched.
\ No newline at end of file
+Finch currently fetches pay statements from the start of two years from the sync time. For example, if a sync happens on 06/01/2022, pay statements from 01/01/2020 thru 06/01/2022 will be fetched.
```

Comment:
We will fetch beyond the "sync time" as time proceeds... it's not like because you connected a token today we only gather up until today. A month from now the job running for that token will be geetting the recent data. It doesn't just stop/end.